docs/pages/_data/services/elastic.yml

```diff
@@ -18,3 +19,10 @@ managing:
 supportedVersions:
   techExampleVersion: 6.1.3
   techLink: "the [Elastic Stack](https://www.elastic.co/downloads/elasticsearch) and [X-Pack](https://www.elastic.co/downloads/x-pack)"
+
+security:
+  plaintext: false
+  extras: ",
```

Comment:
FWIW it may be possible to do a yaml-style multiline statement here at some point. But this is fine for now. At this point it wouldn't surprise me if those didn't work in metalsmith for some reason.